app/src/main/python/gogdl/launch.py

```
@@ -0,0 +1,284 @@
+import os
+import json
+import sys
+import subprocess
+import time
+from gogdl.dl.dl_utils import get_case_insensitive_name
+from ctypes import *
```

Ruff (0.12.2)

`from ctypes import *` used; unable to detect undefined names

(F403)